src/native.js

```diff
@@ -1,20 +1,12 @@
 import { deepAccess, getBidRequest, logError, triggerPixel, insertHtmlIntoIframe } from './utils';
 import includes from 'core-js/library/fn/array/includes';
 
-export const nativeAdapters = [];
+const CONSTANTS = require('./constants.json');
 
-export const NATIVE_KEYS = {
-  title: 'hb_native_title',
-  body: 'hb_native_body',
-  sponsoredBy: 'hb_native_brand',
-  image: 'hb_native_image',
-  icon: 'hb_native_icon',
-  clickUrl: 'hb_native_linkurl',
-  cta: 'hb_native_cta',
-};
+export const nativeAdapters = [];
 
-export const NATIVE_TARGETING_KEYS = Object.keys(NATIVE_KEYS).map(
-  key => NATIVE_KEYS[key]
+export const NATIVE_TARGETING_KEYS = Object.keys(CONSTANTS.NATIVE_KEYS).map(
+  key => CONSTANTS.NATIVE_KEYS[key]
 );
```

Comment:
unless I'm missing something, this is just copying `CONSTANTS.NATIVE_KEYS`.  Why not just use `Object.assign({}, CONSTANTS.NATIVE_KEYS);` instead?